internal/config/config.go

```diff
@@ -54,6 +56,14 @@ func Init(name string, rootCmd cobra.Command, vip *viper.Viper, configChanged fu
 		vip.AddConfigPath("./")
 		vip.AddConfigPath("$HOME/")
 		vip.AddConfigPath("/etc/")
+
+		// Add the executable path to the config search path.
+		binPath, err := os.Executable()
+		if err != nil {
```

Comment:
Fun fact, you can (and I suggest ;) that you use:
`if binPath, err := os.Executable(); err != nil {`

And yes, before you ask it, `binPath` scope will be available in the `else` close.

That was the go trick of the day ;)